yarn.lock

```diff
@@ -2,6 +2,358 @@
 # yarn lockfile v1
```

Comment:
I think we should have either a `package-lock.json` or `yarn.lock`. I think they store `node_modules` slightly differently and if someone adds a dependency with `npm` the next `yarn` user won't have it in their `yarn.lock`.

Any thoughts @gka?

Reply:
I second we should remove yarn.lock. Out of habit, I sometimes accidentally use `yarn`, but we should really stick to one package manager.